### ISSUE TITLE
Failed to record mic

### ISSUE THREAD

**Issue by SigmaBonder** [chenxiaolong/BCR#191]
```
--------- beginning of main
12-08 23:31:30.863  4351  4351 I om.chiller3.bcr: Late-enabling -Xcheck:jni
12-08 23:31:30.894  4351  4351 D CompatibilityChangeReporter: Compat change id reported: 171979766; UID 10040; state: ENABLED
12-08 23:31:30.908  4351  4351 W ziparchive: Unable to open '/system/priv-app/CallRecorder/CallRecorder.dm': No such file or directory
12-08 23:31:30.908  4351  4351 W ziparchive: Unable to open '/system/priv-app/CallRecorder/CallRecorder.dm': No such file or directory
12-08 23:31:30.955  4351  4351 V GraphicsEnvironment: ANGLE Developer option for 'com.chiller3.bcr' set to: 'default'
12-08 23:31:30.955  4351  4351 V GraphicsEnvironment: ANGLE GameManagerService for com.chiller3.bcr: false
12-08 23:31:30.955  4351  4351 V GraphicsEnvironment: Neither updatable production driver nor prerelease driver is supported.
12-08 23:31:30.957  4351  4351 D NetworkSecurityConfig: No Network Security Config specified, using platform default
12-08 23:31:30.957  4351  4351 D NetworkSecurityConfig: No Network Security Config specified, using platform default
12-08 23:31:30.975  4351  4381 D vulkan  : searching for layers in '/system/lib64'
12-08 23:31:30.975  4351  4381 D vulkan  : searching for layers in '/system_ext/lib64'
12-08 23:31:31.016  4351  4351 W om.chiller3.bcr: Accessing hidden method Landroid/view/View;->computeFitSystemWindows(Landroid/graphics/Rect;Landroid/graphics/Rect;)Z (unsupported, reflection, allowed)
12-08 23:31:31.016  4351  4351 W om.chiller3.bcr: Accessing hidden method Landroid/view/ViewGroup;->makeOptionalFitsSystemWindows()V (unsupported, reflection, allowed)
12-08 23:31:31.032  4351  4351 D CompatibilityChangeReporter: Compat change id reported: 210923482; UID 10040; state: ENABLED
12-08 23:31:31.152  4351  4381 E cutils-trace: Error opening trace file: No such file or directory (2)
12-08 23:31:58.269  4351  4351 D CompatibilityChangeReporter: Compat change id reported: 78294732; UID 10040; state: ENABLED
12-08 23:32:09.382  4351  4351 W OnBackInvokedCallback: OnBackInvokedCallback is not enabled for the application.
12-08 23:32:09.382  4351  4351 W OnBackInvokedCallback: Set 'android:enableOnBackInvokedCallback="true"' in the application manifest.
12-08 23:32:09.395  4351  4351 D CompatibilityChangeReporter: Compat change id reported: 171228096; UID 10040; state: ENABLED
12-08 23:32:10.934  4351  4351 W OnBackInvokedCallback: OnBackInvokedCallback is not enabled for the application.
12-08 23:32:10.934  4351  4351 W OnBackInvokedCallback: Set 'android:enableOnBackInvokedCallback="true"' in the application manifest.
12-08 23:32:26.103  4537  4537 I om.chiller3.bcr: Late-enabling -Xcheck:jni
12-08 23:32:26.153  4537  4537 D CompatibilityChangeReporter: Compat change id reported: 171979766; UID 10040; state: ENABLED
12-08 23:32:26.169  4537  4537 W ziparchive: Unable to open '/system/priv-app/CallRecorder/CallRecorder.dm': No such file or directory
12-08 23:32:26.169  4537  4537 W ziparchive: Unable to open '/system/priv-app/CallRecorder/CallRecorder.dm': No such file or directory
12-08 23:32:26.230  4537  4537 V GraphicsEnvironment: ANGLE Developer option for 'com.chiller3.bcr' set to: 'default'
12-08 23:32:26.230  4537  4537 V GraphicsEnvironment: ANGLE GameManagerService for com.chiller3.bcr: false
12-08 23:32:26.231  4537  4537 V GraphicsEnvironment: Neither updatable production driver nor prerelease driver is supported.
12-08 23:32:26.234  4537  4537 D NetworkSecurityConfig: No Network Security Config specified, using platform default
12-08 23:32:26.235  4537  4537 D NetworkSecurityConfig: No Network Security Config specified, using platform default
12-08 23:32:27.612  4537  4537 I RecorderThread/253: Created thread for call: null
12-08 23:32:27.635  4537  4537 D CompatibilityChangeReporter: Compat change id reported: 160794467; UID 10040; state: ENABLED
12-08 23:32:27.643  4537  4558 D RecorderThread/253: Starting log file (1.26.r3.g5a4c510)
12-08 23:32:27.645  4537  4558 D RecorderThread/253: Creating 20221208_233227.622723+0100_mic.log with MIME type text/plain in file:///storage/emulated/0/Android/data/com.chiller3.bcr/files
12-08 23:32:27.698  4537  4558 I RecorderThread/253: Recording thread started
12-08 23:32:27.701  4537  4558 D FilenameTemplate: Loading default filename template
12-08 23:32:27.702  4537  4558 D FilenameTemplate: Filename template: {filename.5.prefix=_, filename.4.prefix=_, filename.3.prefix=_, filename.2.prefix=_sim, filename.1.prefix=_, filename.5.text=${contact_name}, filename.4.text=${caller_name}, filename.3.text=${phone_number}, filename.2.text=${sim_slot}, filename.1.text=${direction}, filename.0.text=${date}}
12-08 23:32:27.706  4537  4558 D FilenameTemplate: Loaded filename components: [Component(text=${date}, default=null, prefix=null, suffix=null), Component(text=${direction}, default=null, prefix=_, suffix=null), Component(text=${sim_slot}, default=null, prefix=_sim, suffix=null), Component(text=${phone_number}, default=null, prefix=_, suffix=null), Component(text=${caller_name}, default=null, prefix=_, suffix=null), Component(text=${contact_name}, default=null, prefix=_, suffix=null)]
12-08 23:32:27.707  4537  4558 E RecorderThread/253: Error during recording
12-08 23:32:27.707  4537  4558 E RecorderThread/253: java.lang.NullPointerException
12-08 23:32:27.707  4537  4558 E RecorderThread/253: 	at com.chiller3.bcr.RecorderThread.run(RecorderThread.kt:237)
12-08 23:32:27.707  4537  4558 I RecorderThread/253: Recording thread completed
12-08 23:32:27.707  4537  4558 D RecorderThread/253: Stopping log file
```
Debug build from `mic` branch.

**Comment by chenxiaolong** [comment]
Ah crap, I probably broke it when rebasing on top of `v1.26` with the filename template changes. I'll look into it.

**Comment by chenxiaolong** [comment]
Fixed in fc1af4152d7368e28d26cfe07b415e6f61b636f4. `mic` branch has been updated.
